data-access-analysis-tools/index.rst

```diff
@@ -36,9 +36,19 @@ Scroll down to the bottom of this page for more information about the future RSP
 Rubin Science Platform (RSP)
 ============================
 
-The Rubin Science Platform (RSP) provides access to Rubin Observatory data products.
-The :ref:`Tools-RSP-Notebook` enables programmatic interaction with, and analysis of, data products in a python environment.
-The :ref:`Tools-RSP-Portal` provides interactive query, data discovery, and visualization tools.
+The Rubin Science Platform (RSP) provides access to Rubin Observatory data products via three aspects: Portal, Notebooks, and API.
+All three aspects enable users to query, retrieve, and visualize the image and catalog data, but in different ways, 
+and with different analysis options.
+
+The :ref:`Tools-RSP-Portal` provides a GUI (graphical user interface) that enables access and visualization without ADQL or python, 
+however, analysis tools such as trend regression are not currently available.
```

Comment:
Does this wording ("not currently available") imply that these analysis tools will be at some point available in the future? Would this be something to address in the "Future of RSP" section?

Reply:
It did -- the "currently" has been removed as per: https://lsstc.slack.com/archives/CAS7Y9ADS/p1690004693108669